rts/Sim/Objects/SolidObject.h

```diff
@@ -383,6 +383,7 @@ class CSolidObject: public CWorldObject {
 	LocalModel localModel;
 	CollisionVolume collisionVolume;
 	CollisionVolume selectionVolume;
+	float selectionRadius = 0.0f;
```

Comment:
Isn't there some neat convenience function to check `CollisionVolume`'s distance to a plane? It would be good not to converge on a worse solution (for example only being able to do a sphere would fail hard for elongated units).

Reply:
Unfortunately only a distance to point exists